k-distribution/include/builtin/domains.k

```diff
@@ -590,6 +590,98 @@ module STRING-BUFFER
   imports STRING-BUFFER-SYMBOLIC
 endmodule
 
+module BYTES-SYNTAX
+  syntax Bytes [hook(BYTES.Bytes)]
+
+  syntax Bytes ::= ".Bytes" [function, hook(BYTES.empty), impure]
+  syntax Endianness ::= "LE" [klabel(littleEndianBytes)]
+                      | "BE" [klabel(bigEndianBytes)]
+  syntax Signedness ::= "Signed" [klabel(signedBytes)]
+                      | "Unsigned" [klabel(unsignedBytes)]
+  syntax Int ::= Bytes2Int(Bytes, Endianness, Signedness) [function, hook(BYTES.bytes2int)]
+  syntax Bytes ::= Int2Bytes(Int, Int, Endianness) [function, hook(BYTES.int2bytes)]
+                 | Int2Bytes(Int, Endianness, Signedness) [function, klabel(Int2BytesNoLen)]
```

Comment:
This function (KLabel `Int2BytesNoLen`) will fail if someone imports the `BYTES-SYNTAX` module, even though the rest will succeed.